src/pipeline.js

```diff
@@ -215,8 +214,16 @@ class Pipeline {
    * @returns {Array.<Annotator>} annotators - those set for this pipeline
    */
   _getAnnotators() {
-    return this._getAnnotatorsKeys()
-      .map(annotatorKey => ANNOTATORS_BY_KEY[annotatorKey]);
+    return this._getAnnotatorsKeys().reduce((acc, annotatorKey) => {
+      if (
+        Object.prototype.hasOwnProperty.call(ANNOTATORS_BY_KEY, annotatorKey)
+      ) {
+        acc.push(ANNOTATORS_BY_KEY[annotatorKey]);
```

Comment:
I’d expect a pure function being passed as reducer.  Instead of push you can return acc.concat([newelem]);  (sorry the syntax, i’m With my phone) 🙃